### PR TITLE
ign -> gz Migrate Ignition Headers : sdformat

### PR DESCRIPTION
**Pull request by methylDragon**
See: gazebo-tooling/release-tools#784

### CODE REVIEW

**Comment by scpeters** [comment]
needs release of https://github.com/gazebosim/gz-math/pull/483

**Comment by scpeters** [comment]
@osrf-jenkins run tests please

File: include/sdf/Actor.hh [hunk 1]
```diff
@@ -175,11 +175,11 @@ namespace sdf
 
     /// \brief Get the pose to be reached.
     /// \return Pose to be reached.
-    public: ignition::math::Pose3d Pose() const;
+    public: gz::math::Pose3d Pose() const;
```
Comment (chapulina) [inline include/sdf/Actor.hh:178]: Can the include also be updated?

**Comment by scpeters** [comment]
we also need to update the minimum version of ignition-math6 needed once we start using `gz/*` and `gz::`

File: include/sdf/Actor.hh (hunk 1 above)
Comment (methylDragon) [reply include/sdf/Actor.hh:178]: Forgot to note that it's been updated!

**Comment by scpeters** [comment]
> we also need to update the minimum version of ignition-math6 needed once we start using `gz/*` and `gz::`

added in https://github.com/gazebosim/sdformat/pull/1118/commits/9624a0ceabe3df324a1bacb32d25fb4b949e2b39

File: src/parser_urdf.cc [hunk 2]
```diff
@@ -3484,7 +3484,7 @@ void ReduceSDFExtensionPluginFrameReplace(
         // create new offset xml blocks
         xyzKey = new TiXmlElement("xyzOffset");
         rpyKey = new TiXmlElement("rpyOffset");
-        correctedOffsetKey = new TiXmlElement("ignition::corrected_offsets");
+        correctedOffsetKey = new TiXmlElement("gz::corrected_offsets");
```
Comment (scpeters) [inline src/parser_urdf.cc:3487]: these changes to the element name should be reverted

File: src/parser_urdf.cc [hunk 3]
```diff
@@ -3466,12 +3466,12 @@ void ReduceSDFExtensionPluginFrameReplace(
...
           urdf::Vector3 rpy = ParseVector3(rpyKey);
           _reductionTransform.Rot() =
-            ignition::math::Quaterniond::EulerToQuaternion(rpy.x, rpy.y, rpy.z);
+            gz::math::Quaterniond::EulerToQuaternion(rpy.x, rpy.y, rpy.z);
           // remove xyzOffset and rpyOffset
           (*_blobIt)->RemoveChild(rpyKey);
         }
         TiXmlNode* correctedOffsetKey =
-            (*_blobIt)->FirstChild("ignition::corrected_offsets");
+            (*_blobIt)->FirstChild("gz::corrected_offsets");
```
Comment (scpeters) [inline src/parser_urdf.cc:3474]: these changes to the element name should be reverted

File: src/parser_urdf.cc (hunk 2 above)
Comment (nkoenig) [reply src/parser_urdf.cc]: [fb6d5f8](https://github.com/gazebosim/sdformat/pull/1118/commits/fb6d5f826605b46774720042e9fa6110baee5090)

File: src/parser_urdf.cc (hunk 3 above)
Comment (nkoenig) [reply src/parser_urdf.cc]: [fb6d5f8](https://github.com/gazebosim/sdformat/pull/1118/commits/fb6d5f826605b46774720042e9fa6110baee5090)

**Review by nkoenig — APPROVED** [review]

**Comment by nkoenig** [comment]
@scpeters, is this safe to merge?

**Comment by scpeters** [comment]
> @scpeters, is this safe to merge?

just made a last few changes I hadn't yet committed. We can merge when CI is clean

**Review by scpeters — APPROVED** [review]

**Pull request merged**